<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainFrame">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="Layout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="FirstName">
          <property name="font">
           <font>
            <family>Noto Serif Tamil SemiBold</family>
            <pointsize>36</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>First name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SecondName">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Second name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="generate">
        <property name="text">
         <string>Generate new Identity</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>27</height>
    </rect>
   </property>
   <addaction name="settings_button"/>
  </widget>
  <action name="settings_button">
   <property name="text">
    <string>settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
